android/gradlew.bat

```diff
@@ -1,84 +0,0 @@
-@if "%DEBUG%" == "" @echo off

```

Comment:
Shouldn't we also remove `android/gradlew`?

Reply:
That was my initial approach, but it seems to be used as a marker by RN to check if it has been installed or not, so I just left it. Maybe we could just change the contents to point to Nix's gradle (with a comment), so it's easier to figure out what's happening? Hopefully RN will play along with that.

Reply:
Could work. I never fully got the point of that file. First time I hear it's required by RN.